include/util/guidance/name_announcements.hpp

```diff
@@ -70,7 +70,9 @@ template <typename StringView> inline auto decompose(const StringView &lhs, cons
     const auto trim = [](StringView view) {
         // we compare suffixes based on this value, it might break UTF chars, but as long as we are
         // consistent in handling, we do not create bad results
-        std::string str = boost::to_lower_copy(view.to_string());
+        std::string str(view);
```

Comment:
nit.

seems to be a little bit faster to do smth like:
```
std::string str;
str.reserve(view.size());
for (auto c : view) {
  str += std::tolower(ch);
}
```
(it probably can be done the same way with some STL algorithms too)